modules/nrfxlib/nrf_802154/sl/platform/nrf_802154_platform_timestamper.c

```diff
@@ -75,6 +75,93 @@ static int32_t m_timestamp_cc_channel;
 #error PPIB_G1_TS_CHANNEL is required to be in the [8..15] range
 #endif
 
+/* Ensure something similar to this is present in the DT:
+ * &dppic130 {
+ *	owned-channels = <3>;
+ *	source-channels = <3>;
+ * };
+ * where 3 corresponds to DPPIC_G1_TS_CHANNEL.
+ */
+#define _TS_DT_CHECK_DPPIC_G1_CHANNEL(node_id, prop, idx) \
+	(DT_PROP_BY_IDX(node_id, prop, idx) == DPPIC_G1_TS_CHANNEL) ||
+
+#define TS_DT_HAS_RESERVED_DPPIC_G1_CHANNEL		\
+	COND_CODE_1(DT_NODE_HAS_PROP(DT_NODELABEL(dppic130), source_channels), \
+		    (DT_FOREACH_PROP_ELEM(DT_NODELABEL(dppic130), \
+		    			  source_channels, \
+					  _TS_DT_CHECK_DPPIC_G1_CHANNEL) false), \
+		    (false))
+
+BUILD_ASSERT(TS_DT_HAS_RESERVED_DPPIC_G1_CHANNEL,
+	     "The required DPPIC_G1 channel is not reserved");
+
+/* Ensure something similar to this is present in the DT:
+ * &dppic132 {
+ *	owned-channels = <3>;
+ *	sink-channels = <3>;
+ * where 3 corresponds to DPPIC_G2_TS_CHANNEL.
+ * };
```

Comment:
To preserve consistency
```suggestion
 * };
 * where 3 corresponds to DPPIC_G2_TS_CHANNEL.
```